server/chat-commands/info.ts

```diff
@@ -822,12 +822,14 @@ export const commands: ChatCommands = {
 		const modName = target.split(',');
```

Comment:
The `modName` and `targets` stuff looks pretty complicated. I suspect they can be combined into one thing, though.